SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1890,19 +1893,29 @@ void USpatialNetDriver::HandleStartupOpQueueing(const TArray<Worker_OpList*>& In
 	}
 
 	QueuedStartupOpLists.Append(InOpLists);
-	bIsReadyToStart = FindAndDispatchStartupOps(InOpLists);
+	if (IsServer())
+	{
+		bIsReadyToStart = FindAndDispatchStartupOpsServer(InOpLists);
+
+		if (bIsReadyToStart)
+		{
+			// We've found and dispatched all ops we need for startup, trigger BeginPlay()
+			// on the GSM and process the queued ops.  Note that FindAndDispatchStartupOps()
+			// will have notified the Dispatcher to skip the startup ops that we've
+			// processed already.
+			GlobalStateManager->TriggerBeginPlay();
```

Comment:
This could go into the server startup op function but it doesn't really matter.

Reply:
It might be cleaner. However, I wouldn't expect from the function named `FindAndDispatchStartupOpsServer` to call `GlobalStateManager->TriggerBeginPlay();` So will keep it where it is for now.

Reply:
That's fair.